docs/install/any-kubernetes-cluster.md

```diff
@@ -41,7 +41,7 @@ This guide assumes that you want to install an upstream Knative release on a
 Kubernetes cluster. A growing number of vendors have managed Knative offerings;
 see the [Knative Offerings](../knative-offerings.md) page for a full list.
 
-Knative {{< version >}} requires a Kubernetes cluster v1.17 or newer, as well as
+Knative {{< version >}} requires a Kubernetes cluster [v1.18 or newer](https://github.com/knative/community/blob/main/mechanics/RELEASE-VERSIONING-PRINCIPLES.md#knative-eventing-version-table), as well as
```

Comment:
The link should be https://github.com/knative/community/blob/main/mechanics/RELEASE-VERSIONING-PRINCIPLES.md#knative-version-tables

Reply:
Fixed